README.md

```diff
@@ -184,8 +185,8 @@ using `balena ssh <uuid> [serviceName]`.
 #### 1.2 Access Restrictions
 
 There are limits on what a support agent may do with a device they have
-been granted access to. This includes the alteration of variables and 
-configurations (both fleet and device).
+been granted access to. This includes the alteration of variables and
+configurations (both fleet and device) and the ability to reboot the device.
```

Comment:
...and apply upgrades, pin releases, etc etc. How much detail do we want here? Best to keep it vague since it can change.

Reply:
Good point @klutchell , I got the impression from the previous authors that they wanted to include the limitations at the time. They have obviously been extended and the one I thought of off the top of my head was device rebooting. Do you think, even if the list is partial, that it would be good to mention upgrades, release pinning, etc? My vote that even a partial list that hints at restrictions, perhaps ending with "and other things", would be better since the whole point of the masterclass is to dabble in a lot of areas to provide insight that an agent/customer may not know. Thoughts?